docs/conf.py

```diff
@@ -11,12 +11,19 @@
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-#
-# import os
-# import sys
-# sys.path.insert(0, os.path.abspath('.'))
 
 import os
+import sys
+
+sys.path.insert(
+    0,
+    os.path.abspath(
```

Comment:
joining an abspath with something result in an abspath ? Is this strictly needed ?

Reply:
Strictly speaking not, I guess.